1.7/usage/service-discovery/virtual-ip-addresses.md

```diff
@@ -3,9 +3,18 @@ post_title: Using Virtual IP Addresses
 post_excerpt: ""
 layout: docs.jade
 ---
-DC/OS can map traffic from a single Virtual IP (VIP) to multiple IP addresses and ports.
+DC/OS can map traffic from a single Virtual IP (VIP) and port to multiple IP addresses and ports. A VIP contains 2 components:
 
-You can assign a VIP to your application by using the DC/OS Marathon web interface. The values you enter in these fields are translated into the appropriate `portMapping` entry in your application definition. Toggle to `JSON mode` as you create your app to see and edit your application definition.
+ * Private virtual IP address
+ * Service port
```

Comment:
The terminology is confusing, because we refer to it as a service port in the Marathon docs (and it's conceptually the same). Regardless, I'm fine with dropping it.